hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/TopicOwnerCache.java

```diff
@@ -0,0 +1,60 @@
+package pl.allegro.tech.hermes.management.domain.topic;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Multimap;
+import com.google.common.collect.Multimaps;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import pl.allegro.tech.hermes.api.OwnerId;
+import pl.allegro.tech.hermes.api.Topic;
+import pl.allegro.tech.hermes.api.TopicName;
+import pl.allegro.tech.hermes.domain.topic.TopicRepository;
+import pl.allegro.tech.hermes.management.domain.group.GroupService;
+
+import javax.annotation.PostConstruct;
+import java.util.Collection;
+
+@Component
+public class TopicOwnerCache {
```

Comment:
but we have more than one `hermes-management` instance, how will you sync the cache between nodes?

Reply:
do we really need a synchronised cache for that?

Reply:
no, we can do it in background once per X. however in current implementation cache is reloaded only on `hermes-management` start. so if i change topic owner, i need to restart management instances? or am i missing something :)

Reply:
`TopicOwnerCache` is used in `TopicService`(especially line https://github.com/allegro/hermes/pull/886/files/db8bf84063b6f6a9b5c6174bcea6eb09aa031cb2#diff-c14949a99fc0d34090e3953832d9f208R133). So cache is always updated, but it's not limited btw.

Reply:
I know, but it is only updated on single instance :) Assume:

* there are two instances of `hermes-management` (`A` and `B`)
* instance `A` and `B` fill cache on startup
* owner of topic `T` is changed, request was sent to `A`
* `A` saves changes in Zookeeper and updates its in-memory cache
* `B` doesn't know about the change
* you get stale data from `B` until it gets restarted :(

Reply:
I can't belive I fell for that one... I will try to use `ZookeeperInternalNotificationBus` from `hermes-commons` that is currently used in `hermes-frontend` and `hermes-consumers`

Reply:
I think simplest solution would be better - notification bus needs to create another set of watches. How about simple refresh every few minutes? It's not critical.

Reply:
Ok, let's try with refreshing for now. We can move to more consistent cache (by using `ZookeeperInternalNotificationBus`) later if that will be a problem